### ISSUE TITLE
Improved Sample app

### ISSUE THREAD

**Issue by adimiz1** [cloudinary/cloudinary-react-native#31]
## Feature request for Cloudinary React Native SDK
Improved Sample app

## Explain your use case
Need an improved sample app to demonstrates the SDK capabilities

## Describe the problem you’re trying to solve
Give developers an easy way to understand the SDK usage through a sample app

## Do you have a proposed solution?
Build an improved sample app

**Comment by mensonones** [comment]
See: https://github.com/cloudinary/cloudinary-react-native/tree/master/example

**Comment by michal-sordyl-cloudinary** [comment]
@mensonones Would you like to work on improving the sample application?

**Comment by mensonones** [comment]
> @mensonones Would you like to work on improving the sample application?

yes! I can create an example of image upload, or do you have something specific in mind?
